### ISSUE TITLE
Crash with custom Thermopneumatic Processing Plant Recipe

### ISSUE THREAD

**Issue by MuteTiefling** [TeamPneumatic/pnc-repressurized#409]
*For feature requests, just erase this template and clearly describe the feature you'd like to see*

### Minecraft Version
1.12.2 
### Forge Version
14.23.5.2838
### Mod Version
1.12.2-0.11.7-375 
### Describe your problem, including steps to reproduce it

I've made a simple recipe to melt copper items into fluid (Thermal for Copper, NuclearCraft for Molten Copper) and it's crashing. Haven't had any issue with the previous recipes I've made for this machine. 

### Any other comments?

https://paste.dimdev.org/alasorufoq.mccrash

```
import crafttweaker.item.IItemStack;
import crafttweaker.liquid.ILiquidStack;
import mods.pneumaticcraft.thermopneumaticprocessingplant as ThermopneumaticProcessingPlant;
//Thermopneumatic Processing Plant (LiquidInput, ItemInput, Pressure, temp, LiquidOutput)
	//Pressure range: -1.0 to 5.0
	//Temperature Range: 20-2000 K (realistically)

	//Melting snow/ice for water
	ThermopneumaticProcessingPlant.addRecipe(null, <minecraft:snow>, 0.0, 273, <liquid:water> * 500);
	ThermopneumaticProcessingPlant.addRecipe(null, <minecraft:snowball>, 0.0, 273, <liquid:water> * 125);
	ThermopneumaticProcessingPlant.addRecipe(null, <minecraft:packed_ice>, 0.0, 273, <liquid:water> * 2000);
	ThermopneumaticProcessingPlant.addRecipe(null, <minecraft:ice>, 0.0, 273, <liquid:water> * 1000);
	ThermopneumaticProcessingPlant.addRecipe(null, <minecraft:snow_layer>, 0.0, 273, <liquid:water> * 250);

	//Processing ethanol to ethene for plastic production
	ThermopneumaticProcessingPlant.addRecipe(<liquid:ethanol> * 1000, null, 4.0, 443, <liquid:ethene> * 750);

	//Aerogel Processing
	ThermopneumaticProcessingPlant.addRecipe(<liquid:cryotheum> * 1000, <contenttweaker:aerogel_powder>, 4.0, 1000, <liquid:aerogel_slurry> * 1000);

	//Metal Melting	
	//Copper
	ThermopneumaticProcessingPlant.addRecipe(null, <thermalfoundation:material:192>, 1.0, 1357, <liquid:copper> * 16);
	ThermopneumaticProcessingPlant.addRecipe(null, <thermalfoundation:material:64>, 1.0, 1357, <liquid:copper> * 144);
	ThermopneumaticProcessingPlant.addRecipe(null, <thermalfoundation:material:128>, 1.0, 1357, <liquid:copper> * 144);
	ThermopneumaticProcessingPlant.addRecipe(null, <thermalfoundation:storage:0>, 1.0, 1357, <liquid:copper> * 1296);

```

Still investigating on my side. Will update if I figure anything out.

**Comment by MuteTiefling** [comment]
Hrm, something else is going on here. Even my old recipes are causing crashes now... I had thought maybe it was something goofy with the modded resources, but the earliest ones only use vanilla items. I'll try a new world and see. Maybe this one has gotten corrupt.

**Comment by MuteTiefling** [comment]
Ok. Seems it's specifically an issue with recipes that don't have a fluid input. The ethanol and cryotheum recipes listed above still work in my test world and a new world. The ones with no fluid input, only a solid input, crash upon recipe completion.

**Comment by MuteTiefling** [comment]
Have updated Forge to 14.23.5.2847 now and still experiencing the crash. 

Can also reproduce in a new instance with only JEI, CraftTweaker, and PNC. Same recipe, ice to water.

**Comment by desht** [comment]
Yeah, think I see the issue here.  Surprised this hasn't come up before.  Easy fix though.

**Comment by MuteTiefling** [comment]
Maybe nobody's been crazy enough to try do make a custom metal processing system through PNC before :D Trying to use chains of these guys as a makeshift alloy smelter. 

Anyway, thanks for looking into it. :D

**Comment by desht** [comment]
Can you give build 378 from https://jenkins.k-4u.nl/job/PneumaticCraft-Repressurized/ a go?  It should fix your problem.

**Comment by MuteTiefling** [comment]
Works beautifully :D Awesome stuff!

**Comment by desht** [comment]
Fixed in 0.11.8 release
